### PR TITLE
values for Clickup credentials

### PR DESCRIPTION
**Pull request by astreter**

### CODE REVIEW

**Comment by astreter** [comment]
!deploy

**Comment by astreter** [comment]
!deploy

**Comment by astreter** [comment]
!deploy

**Comment by astreter** [comment]
!deploy

File: .helm/templates/secret.yaml [hunk 1]
```diff
@@ -4,8 +4,10 @@ metadata:
   name: {{ include "jiraclick.fullname" . }}
   labels: {{- include "jiraclick.labels" . | nindent 4 }}
 data:
-  clickup_token: {{.Values.clickup.token | b64enc | quote}}
-  clickup_webhooksecret: {{.Values.clickup.webhooksecret | b64enc | quote}}
+  { { - range $key, $value := .Values.clickup } }
```
Comment (kuzaxak) [inline .helm/templates/secret.yaml]:
```suggestion
  {{- range $key, $value := .Values.clickup }}
```

File: .helm/templates/secret.yaml [hunk 2]
```diff
@@ -4,8 +4,10 @@ metadata:
...
+  clickup_{{$key}}_webhooksecret: {{$value.webhooksecret | b64enc | quote}}
+  clickup_{{$key}}_token: {{$value.token | b64enc | quote}}
+  { { - end } }
```
Comment (kuzaxak) [inline .helm/templates/secret.yaml]:
```suggestion
  {{- end }}
```

**Review by kuzaxak — CHANGES_REQUESTED** [review]

File: .helm/templates/secret.yaml (hunk 1 above)
Comment (astreter) [reply .helm/templates/secret.yaml]: fixed

**Comment by astreter** [comment]
!deploy

**Comment by astreter** [comment]
!deploy

File: .helm/values.yaml [hunk 3]
```diff
@@ -39,8 +39,16 @@ service:
...
+  CQYUVTWH2:
+    webhooksecret: ""
+    token: "pk_"
+    list: "10942379"
+    host: "https://api.clickup.com/api/v2"
+  CQTG74FUH:
+    webhooksecret: ""
+    token: "pk_"
+    list: "10942379"
+    host: "https://api.clickup.com/api/v2"
```
Comment (kuzaxak) [inline .helm/values.yaml:51]: I think you want to place it to config_content. config_content stored in secret.yaml

File: pkg/config/config.go [hunk 4]
```diff
@@ -52,6 +52,18 @@ func NewConfig(path string) (*Config, error) {
...
+	}
+	if err := viper.BindEnv("clickup.CQYUVTWH2.token"); err != nil {
+		return nil, err
+	}
+	if err := viper.BindEnv("clickup.CQTG74FUH.webhooksecret"); err != nil {
+		return nil, err
+	}
+	if err := viper.BindEnv("clickup.CQTG74FUH.token"); err != nil {
+		return nil, err
+	}
```
Comment (kuzaxak) [inline pkg/config/config.go]: Please read from file. Do not hadcode `CQYUVTWH2` in go code

**Comment by astreter** [comment]
!deploy

File: pkg/config/config.go (hunk 4 above)
Comment (astreter) [reply pkg/config/config.go]: done

File: .helm/values.yaml (hunk 3 above)
Comment (astreter) [reply .helm/values.yaml:51]: fixed

**Pull request merged**